packages/webapp/src/utils/combined-case-constants.js

```diff
@@ -1,12 +1,224 @@
+  COMBINED_CASE_CLIENTS_ORGANISATION_NAME: reusedRoutePath(baseUrl, CLIENTS_ORGANISATION_NAME),
+  COMBINED_CASE_DEFRA_ACCOUNT_NOT_LINKED: reusedRoutePath(baseUrl, DEFRA_ACCOUNT_NOT_LINKED),
+  COMBINED_CASE_NON_UK_ADDRESS: reusedRoutePath(baseUrl, NON_UK_ADDRESS),
+  COMBINED_CASE_UK_ADDRESS: reusedRoutePath(baseUrl, UK_ADDRESS),
+  COMBINED_CASE_CHECK_APPLICANT_INFORMATION: reusedRoutePath(baseUrl, CHECK_APPLICANT_INFORMATION),
+  COMBINED_CASE_CLIENTS_EMAIL_ADDRESS: reusedRoutePath(baseUrl, CLIENTS_EMAIL_ADDRESS),
+  COMBINED_CASE_UPLOAD_WRITTEN_AUTHORISATION: reusedRoutePath(baseUrl, UPLOAD_WRITTEN_AUTHORISATION),
+  COMBINED_CASE_CHECK_WRITTEN_AUTHORISATION_FILE: reusedRoutePath(baseUrl, CHECK_WRITTEN_AUTHORISATION_FILE),
+  COMBINED_CASE_CLIENTS_PHONE_NUMBER: reusedRoutePath(baseUrl, CLIENTS_PHONE_NUMBER),
+  COMBINED_CASE_ADD_GRID_REFERENCE: reusedRoutePath(baseUrl, ADD_GRID_REFERENCE),
+  COMBINED_CASE_NEED_ADD_ALL_LEGAL_FILES: reusedRoutePath(baseUrl, NEED_ADD_ALL_LEGAL_FILES),
+  COMBINED_CASE_NEED_LEGAL_AGREEMENT: reusedRoutePath(baseUrl, NEED_LEGAL_AGREEMENT),
+  COMBINED_CASE_UPLOAD_LEGAL_AGREEMENT: reusedRoutePath(baseUrl, UPLOAD_LEGAL_AGREEMENT),
+  COMBINED_CASE_CHECK_LEGAL_AGREEMENT: reusedRoutePath(baseUrl, CHECK_LEGAL_AGREEMENT),
+  COMBINED_CASE_CHECK_LEGAL_AGREEMENT_FILES: reusedRoutePath(baseUrl, CHECK_LEGAL_AGREEMENT_FILES),
+  COMBINED_CASE_NEED_ADD_ALL_RESPONSIBLE_BODIES: reusedRoutePath(baseUrl, NEED_ADD_ALL_RESPONSIBLE_BODIES),
+  COMBINED_CASE_NEED_ADD_ALL_PLANNING_AUTHORITIES: reusedRoutePath(baseUrl, NEED_ADD_ALL_PLANNING_AUTHORITIES),
+  COMBINED_CASE_ADD_RESPONSIBLE_BODY_CONVERSATION_COVENANT: reusedRoutePath(baseUrl, ADD_RESPONSIBLE_BODY_CONVERSATION_COVENANT),
+  COMBINED_CASE_CHECK_RESPONSIBLE_BODIES: reusedRoutePath(baseUrl, CHECK_RESPONSIBLE_BODIES),
+  COMBINED_CASE_ANY_OTHER_LANDOWNERS: reusedRoutePath(baseUrl, ANY_OTHER_LANDOWNERS),
+  COMBINED_CASE_LANDOWNER_INDIVIDUAL_ORGANISATION: reusedRoutePath(baseUrl, LANDOWNER_INDIVIDUAL_ORGANISATION),
+  COMBINED_CASE_HABITAT_PLAN_LEGAL_AGREEMENT: reusedRoutePath(baseUrl, HABITAT_PLAN_LEGAL_AGREEMENT),
+  COMBINED_CASE_ADD_LANDOWNER_INDIVIDUAL: reusedRoutePath(baseUrl, ADD_LANDOWNER_INDIVIDUAL),
+  COMBINED_CASE_ADD_LANDOWNER_ORGANISATION: reusedRoutePath(baseUrl, ADD_LANDOWNER_ORGANISATION),
+  COMBINED_CASE_CHECK_LANDOWNERS: reusedRoutePath(baseUrl, CHECK_LANDOWNERS),
+  COMBINED_CASE_ENHANCEMENT_WORKS_START_DATE: reusedRoutePath(baseUrl, ENHANCEMENT_WORKS_START_DATE),
+  COMBINED_CASE_UPLOAD_HABITAT_PLAN: reusedRoutePath(baseUrl, UPLOAD_HABITAT_PLAN),
+  COMBINED_CASE_CHECK_HABITAT_PLAN_FILE: reusedRoutePath(baseUrl, CHECK_HABITAT_PLAN_FILE),
+  COMBINED_CASE_HABITAT_ENHANCEMENTS_END_DATE: reusedRoutePath(baseUrl, HABITAT_ENHANCEMENTS_END_DATE),
+  COMBINED_CASE_ADD_PLANNING_AUTHORITY: reusedRoutePath(baseUrl, ADD_PLANNING_AUTHORITY),
+  COMBINED_CASE_CHECK_PLANNING_AUTHORITIES: reusedRoutePath(baseUrl, CHECK_PLANNING_AUTHORITIES)
+}
+
+export {
```

Comment:
Just checking the reason for this being `export { ... }` when it was previously `export default { ... }`? If we reverted to `export default` then would that mean we could just use `import combinedCaseConstants` elsewhere instead of `import * as combinedCaseConstants`?